api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -666,9 +674,185 @@ public void shortenedStackTrace_shouldReturnNullIfStackTraceIsNull() {
 	}
 	
 	/**
-	 * @see OpenmrsUtil#nullSafeEqualsIgnoreCase(String,String)
+	 * @see OpenmrsUtil#folderContains(File, String)
+	 */
+	@Test
+	public void folderContains_shouldReturnTrueWhenFolderContainsFileWithGivenName() {
+		String path = System.getProperty("user.dir");
+		File folder = new File(path+"/folder");
+		folder.mkdir();
+		File file = new File(path+"/folder/file");
```

Comment:
Will this file path work on all OSs! Will it work properly on windows? Can you use Java's abstraction and get system path symbols ie "/" or "\"

Reply:
You are right . I will change with System file separator now.